app/controllers/search_controller.rb

```diff
@@ -19,6 +28,8 @@ def build_params
   end
 
   def search_params
-    params.permit(:sort_by, :order, :collection_id, :q, :page, :deep)
+    params.require(:data).permit(:sort_by, :order,
+                                 :collection_id, :q, :page,
+                                 :deep, :institution_id)
```

Comment:
I'm not sure about other requests coming in, but the ajax modified above sends `sort_id` rather than `sort_by` (not sure if either is used here), no longer sends `deep`, and also doesn't send `page` - possibly some of them are no longer used?